app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -70,6 +70,8 @@ CONFIG_WINSTREAM_CONSOLE=n
 
 CONFIG_INTEL_ADSP_IPC=y
 
+CONFIG_PROBE=y
```

Comment:
commit title: boards

Reply:
OK, thank you fixed

Reply:
btw, the commit still has the typo (updating the pull request title won't change the commit).